File: LeetCode Solutions/2414.cpp
class Solution {
public:
    int longestContinuousSubstring(string s) {
        int cnt = 1, ans = 1;
        for (int i = 1; i < s.size(); ++i){
            if (s[i - 1] - 'a' + 1 == s[i] - 'a'){
                
                ++cnt;
            }else{
                ans = max(ans, cnt);
                cnt = 1;
            }
        }
        ans = max(ans, cnt);
        return ans;
    }
};

File: LeetCode Solutions/2414.py
class Solution(object):
    def longestContinuousSubstring(self, s):
        """
        :type s: str
        :rtype: int
        """
        cnt = 1
        ans = 1
        for i in range(1, len(s)):
            if (ord(s[i-1]) - 95 == ord(s[i]) - 96):
                cnt += 1
            else:
                ans = max(ans, cnt)
                cnt = 1
        ans = max(ans, cnt)
        return ans
